src/test/java/api/support/fakes/FakeFeeFineOperationsModule.java

```diff
@@ -0,0 +1,68 @@
+package api.support.fakes;
+
+import static api.support.fakes.Storage.getStorage;
+import static org.folio.circulation.support.http.server.JsonHttpResponse.created;
+import static org.folio.circulation.support.json.JsonPropertyWriter.write;
+import static org.folio.circulation.support.json.JsonPropertyWriter.writeByPath;
+import static org.folio.rest.util.OkapiConnectionParams.OKAPI_TENANT_HEADER;
+
+import java.util.Map;
+
+import io.vertx.core.json.JsonObject;
+import io.vertx.ext.web.Router;
+import io.vertx.ext.web.RoutingContext;
+
+public class FakeFeeFineOperationsModule {
```

Comment:
Does this mimic the effects on an account that the new APIs should have?

Reply:
Yes, partially. It only make an effect on account record, but the new API also deals with account actions.

Reply:
And we have decided not to mimic those as well, hence the removal of the checks for those in the tests?

Which, if I've understood correctly, is the right call.

Reply:
Correct, it is quite complicated and I don't think it make sense to mimic that behavior.

Reply:
Agreed